media-autobuild_suite.bat

```diff
@@ -1579,6 +1579,16 @@ for /f "tokens=2" %%P in ('tasklist /v ^|findstr MABSbat') do set ourPID=%%P
 if exist %build%\compilation_failed del %build%\compilation_failed
 if exist %build%\fail_comp del %build%\compilation_failed
 
+REM Test mklink availability
+set symlinkSupport=""
+mkdir testmklink
+mklink /d linkedtestmklink testmklink
```

Comment:
maybe add an error redirect since I don't think anyone wants to see a random `You do not have sufficient privilege to perform this operation.` during the script

Reply:
Just `2>NUL` ?

Reply:
yeah, I couldn't remember what it was for batch